opentelekomcloud/services/elb/v3/resource_opentelekomcloud_lb_loadbalancer_v3.go

```diff
@@ -224,6 +232,14 @@ func resourceLoadBalancerV3Read(ctx context.Context, d *schema.ResourceData, met
 
 	log.Printf("[DEBUG] Retrieved loadbalancer %s: %#v", d.Id(), lb)
 
+	publicIpInfo := make([]map[string]interface{}, len(lb.PublicIps))
+	for i, v := range lb.PublicIps {
+		publicIpInfo[i] = map[string]interface{}{
+			"public_ip_id":      v.PublicIpID,
+			"public_ip_address": v.PublicIpAddress,
+		}
```

Comment:
You should set other fields from the schema, e.g:
```
		publicIpInfo[i] = map[string]interface{}{
			"public_ip_id":      v.PublicIpID,
			"public_ip_address": v.PublicIpAddress,
			"bandwidth_name": d.Get("public_ip.0.bandwidth_name").(string)
			...
		}
```

or init it with a value from the schema:
```
info := d.Get("public_ip.0").(map[string]interface{})
info["public_ip_id"] = v.publicIpID
...
publicIpInfo[i] = info
```

(or replace `0` with `i` if you want to support a list of public IPs